bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx66.html

```diff
@@ -0,0 +1,182 @@
+            <div class="mzp-c-newsletter">
+              {{ email_newsletter_form(
+                protocol_component=True,
+                include_title=False,
+                spinner_color='#fff',
+                button_class='mzp-c-button')
+              }}
+            </div>
+          </div>
+        </div>
+      </section>
+      {% endif %}
+
+    {% set content_blocking_image='firefox/whatsnew_64/hero.png' if DIR == 'ltr' else 'firefox/whatsnew_64/hero-rtl.png' %}
+    {% call hero(
+     title=_('One log-in. Power and security everywhere.'),
+     class='mzp-has-image mzp-l-reverse t-wn64 show-fxa-default show-fxa-supported-signed-out',
+     image_url=content_blocking_image,
+     include_highres_image=true,
+     include_cta=True
+    ) %}
+    <p class="show-fxa-supported-signed-out show-fxa-default">
+      <a href="https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64" class="mzp-c-button mzp-t-download">{{_('Create a Firefox Account') }}</a><br>
+      <span class="wn64-signin">{{ _('Have an account?') }} <a href="https://accounts.firefox.com/?action=email&service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64">{{ _('Log in') }}</a></span>
+    </p>
+    {% endcall %}
+    {% call hero(
+     title=_('It shouldn’t be hard to own your life online.'),
+     desc=_('That’s why Firefox gives more power to you with every update.'),
+     class='mzp-has-image mzp-l-reverse t-wn64 show-fxa-supported-signed-in',
+     image_url=content_blocking_image,
+     include_highres_image=true
+    ) %}
+    {% endcall %}
+
+    <div class="mzp-l-content">
+      <div class="wn66-benefit wn66-send">
+        {{ high_res_img('firefox/whatsnew_66/fxa-send.png', {'alt': '', 'width': '590', 'height': '590'}) }}
+        <h3>{{_('Take total control of the files you share ')}}</h3>
+        <p>{{_('Feel like a secret agent when you send files online, with a private, encrypted link that ‘self-destructs.’')}}</p>
+        <p><a class="mzp-c-cta-link" data-app="send" href="
+          https://send.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=wnp66&utm_content=try-send&entrypoint=mozilla.org-whatsnew66">{{_('Try Firefox Send')}}</a></p>
+      </div>
+
+      <div class="wn66-benefit wn66-lockbox">
+        {{ high_res_img('firefox/whatsnew_64/fxa-lockbox.png', {'alt': '', 'width': '590', 'height': '590'}) }}
+        <h3>{{_('Remember your passwords in Lockbox')}}</h3>
+        <p>{{_('Discover the web’s best content, and absorb it anytime – even offline – on any device. Pocket’s listen feature will even read any article aloud to you. Get it all from the Firefox toolbar or the Pocket app.')}}</p>
+        <p><a class="mzp-c-button wn66-benefit-link" data-app="lockbox" href="#">{{_('Get the Lockbox App')}}</a></p>
```

Comment:
Seeing as this opens a modal it should probably be a `<button>`? If there's a good reason why it needs to be a link, we should at least add `role="button"`.